diff --git a/node_modules/@expo/fingerprint/build/sourcer/Bare.js b/node_modules/@expo/fingerprint/build/sourcer/Bare.js
index a290dbe..cdb5e91 100644
--- a/node_modules/@expo/fingerprint/build/sourcer/Bare.js
+++ b/node_modules/@expo/fingerprint/build/sourcer/Bare.js
@@ -166,9 +166,9 @@ async function parseCoreAutolinkingSourcesAsync({ config, reasons, contentsId, p
     for (const [depName, depData] of Object.entries(config.dependencies)) {
         try {
             stripRncoreAutolinkingAbsolutePaths(depData, root);
-            const filePath = (0, Path_1.toPosixPath)(depData.root);
+            const filePath = `${(0, Path_1.toPosixPath)(depData.root)}/package.json`;
             debug(`Adding ${logTag} - ${chalk_1.default.dim(filePath)}`);
-            results.push({ type: 'dir', filePath, reasons });
+            results.push({ type: 'file', filePath, reasons });
             autolinkingConfig[depName] = depData;
         }
         catch (e) {
